tests/fullydistributed_grids/copy_serial_tria_02.with_metis=true.mpirun=1.output

```diff
@@ -0,0 +1 @@
+
```

Comment:
can you output some information about multigrid here? Maybe number of cells per level or something?